### ISSUE TITLE
Implement CustomHandlers

### ISSUE THREAD

**Issue by deusebio** [CGnal/cgnal-core#26]
It would be nice to include in the cgnal-core code base, some custom implementation for the logging handlers that we have done in some of our projects, like:

1. EmailHandler
2. SlackHandler

**Comment by NicolaDonelli** [comment]
I agree with the idea but, at least for what it takes the EmailHandler I implemented myself, it is heavily dependent on the underlying configuration of the SMPT server on the machine that depends itself on some authorization at Domain level... So it is very custom for the given context.
